locales/base/translation.json

```diff
@@ -1682,6 +1682,10 @@
       "title": "Before you swap",
       "body": "Swapping these assets may result in an unfavorable exchange rate. If you lower the amount, you may get a better rate."
     },
+    "missingPriceImpactWarning": {
+      "title": "Before you swap",
+      "body": "We couldn't estimate the price impact of this swap. Proceed carefully."
```

Comment:
Maybe there's a better wording for this?
This message shouldn't be shown normally, but there are some weird cases where this could happen.